content/en/docs/rules/supported-fields.md

```diff
@@ -41,10 +42,11 @@ Name | Type | Description
 `evt.pluginname` | CHARBUF | if the event comes from a plugin, the name of the plugin that generated it. The plugin must be currently loaded.
 `evt.plugininfo` | CHARBUF | if the event comes from a plugin, a summary of the event as formatted by the plugin. The plugin must be currently loaded.
 
-### Field Class: evt (for system calls)
+### Field Class: evt
 
 Event fields applicable to syscall events. Note that for most events you can access the individual arguments/parameters of each syscall via evt.arg, e.g. evt.arg.filename.
 
+
```

Comment:
note for the reviewer: these newlines don't affect the rendering (checked with preview) but they make the one-liner easier to write.